finat/fiat_elements.py

```diff
@@ -333,9 +333,29 @@ def __init__(self, cell, degree, variant=None):
         super(RaviartThomas, self).__init__(FIAT.RaviartThomas(cell, degree, variant=variant))
 
 
+class TrimmedSerendipityFace(VectorFiatElement):
+    def __init__(self, cell, degree):
+        super(TrimmedSerendipityFace, self).__init__(FIAT.TrimmedSerendipityFace(cell, degree))
+
+
+class TrimmedSerendipityDiv(VectorFiatElement):
+    def __init__(self, cell, degree):
+        super(TrimmedSerendipityDiv, self).__init__(FIAT.TrimmedSerendipityDiv(cell, degree))
+
+
+class TrimmedSerendipityEdge(VectorFiatElement):
+    def __init__(self, cell, degree):
+        super(TrimmedSerendipityEdge, self).__init__(FIAT.TrimmedSerendipityEdge(cell, degree))
+
+class TrimmedSerendipityCurl(VectorFiatElement):
+    def __init__(self, cell, degree):
+        super(TrimmedSerendipityCurl, self).__init__(FIAT.TrimmedSerendipityCurl(cell, degree))
+
+
 class BrezziDouglasMarini(VectorFiatElement):
     def __init__(self, cell, degree, variant=None):
-        super(BrezziDouglasMarini, self).__init__(FIAT.BrezziDouglasMarini(cell, degree, variant=variant))
+        #super(BrezziDouglasMarini, self).__init__(FIAT.BrezziDouglasMarini(cell, degree, variant=variant))
+        super(BrezziDouglasMarini, self).__init__(FIAT.BrezziDouglasMarini(cell, degree))
```

Comment:
This part should be dropped. I think it is because the matching FIAT branch doesn't have these variants yet.

Reply:
Agree.